test/micro/org/openjdk/bench/java/lang/Characters.java

```diff
@@ -48,6 +49,29 @@ public class Characters {
     @Param({"9", "65", "97", "223", "430"})
     private int codePoint;
 
+    @Param("500")
+    private int size;
+
+    private char[] chars;
+    private char[] res;
+
+    @Setup
+    public void setup() {
+        Random r  = new Random(0);
+        chars     = new char[size];
+        res       = new char[size];
+        for (int i = 0; i < size; i++) {
+            chars[i] = (char)r.nextInt(Character.MAX_VALUE + 1);
+        }
+    }
+
+    @Benchmark
+    public void reverseBytes() {
```

Comment:
I'm not sure we want to be adding that benchmark to this class. Could you move to a different class that will test exclusively `reverseBytes` on `char[]`? You can then move the code from `test/micro/org/openjdk/bench/java/lang/Shorts.java` into that same class or a similarly named class for `short[]`.

Reply:
Not sure if I understand you correctly.
The test is for reverseBytes of a char, not char[]. And it's quite similar as existing test in for integer at: https://github.com/openjdk/jdk/blob/master/test/micro/org/openjdk/bench/java/lang/Integers.java#L171

Reply:
I meant put this `reverseBytes` benchmark in a different class, but given it's how it's done in https://github.com/openjdk/jdk/blob/master/test/micro/org/openjdk/bench/java/lang/Integers.java#L171, please ignore this comment.